integrations/acquisition/covid_hosp/facility/test_scenarios.py

```diff
@@ -19,48 +19,37 @@
 
 NEWLINE="\n"
 
-class AcquisitionTests(unittest.TestCase):
+class AcquisitionTests(CovidHospTestCase):
 
   def setUp(self):
     """Perform per-test setup."""
 
     # configure test data
     self.test_utils = UnitTestUtils(__file__)
 
-    # use the local instance of the Epidata API
-    Epidata.BASE_URL = 'http://delphi_web_epidata/epidata/api.php'
-
-    # use the local instance of the epidata database
-    secrets.db.host = 'delphi_database_epidata'
-    secrets.db.epi = ('user', 'pass')
-
     # clear relevant tables
-    with Database.connect() as db:
-      with db.new_cursor() as cur:
-        cur.execute('truncate table covid_hosp_facility')
-        cur.execute('truncate table covid_hosp_facility_key')
-        cur.execute('truncate table covid_hosp_meta')
+    # TODO: get these names dynamically from the YAML file once all are available
+    super().setUp(Database(), [
+      'covid_hosp_facility',
+      'covid_hosp_facility_key',
+      'covid_hosp_meta'
+    ])
```

Comment:
with my other suggested changes to the base `CovidHospTestCase`, all of this `setUp()` method can be replaced with just:
```
  db_class = Database
  test_util_context = __file__
```
thereby getting `super().setup()` for free